site/_posts/2020-11-15-how-to-securely-implement-oauth-rails.md

```diff
@@ -0,0 +1,262 @@
+---
+layout: blog-post
+title: How to securely implement OAuth in Ruby on Rails 
+description: A detailed overview of securely integrating an Ruby on Rails application with an OAuth provider using the OAuth Authorization Code Grant  
+author: Brett Guy 
+image: blogs/fusionauth-example-angular/oauth-angular-fusionauth.png
+category: blog
+tags: client-ruby
+excerpt_separator: "<!--more-->"
+---
+
+In this tutorial, we'll walk through setting up a basic Ruby on Rails app to securely authenticate with an OAuth2 server using the authorization code grant.
+
+<!--more-->
+
+Many Rails applications traditionally handle authentication, authorization, and user-management within the framework itself. With FusionAuth, we are able to separate those concerns from our application.
+At the same time, we get the benefit of a complete identity solution capable of handling registration, login, MFA, passwordless, SSO, social-logins, and much more.
```

Comment:
social-logins -> social login

Reply:
Also worth pointing out that you can integrate with lots of COTS and OSS in case you want to have users integrate with a forum, etc.